src/applications/vaos/utils/data.js

```diff
@@ -44,6 +45,29 @@ function getUserMessage(data) {
   return `${label}: ${data.reasonAdditionalInfo}`;
 }
 
+// When useVSP is true, the clinic data is using the FHIR format. The clinic id format is:
+// urn:va:healthcareservice:983:983:308
+// where the last 3 tokens represent the site id, facility id, and clinic id.
+function getToken(clinic, index) {
+  return clinic.identifier[0].value.split(':')[index];
+}
+
+function getSiteCode(clinic, useVSP) {
+  if (clinic) {
+    return useVSP
```

Comment:
Our transformer for HealthcareService creates those URN identifiers: https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/applications/vaos/services/healthcare-service/transformers.js#L20

So I think you could skip the feature toggle check entirely in this PR and always pull the ids you need with this `getToken` call.

Reply:
Done.